pages/checkout/access.tsx

```diff
@@ -79,13 +69,16 @@ const CheckoutAccessView: NextPage = () => {
       setError(false)
       try {
         setIsLoading(true)
-        await HttpClient.post<Collibra.WorkflowInstance>("/api/workflows", {
-          body: {
-            assetId: checkoutData.asset?.id,
-            termsAccepted: checkoutData.terms?.termsAccepted,
-            description: checkoutData.access?.description,
-          },
-        })
+
+        if (config.ACCESS_COLLIBRA_WORKFLOW) {
```

Comment:
after having tested this locally, i think we should reverse the logic here. with the currently proposed implementation, we have to opt in to trigger the collibra workflow. this means that this will change the behaviour of prod, which we don't want feature flags to do, unless toggled.

Reply:
additionally, the variable name should be changed to reflect that it's opt in. so instead of `ACCESS`, you could name it something like `BLOCK` or `PREVENT` 😄